### ISSUE TITLE
Trying to delete a member through Ghost admin fails with 2 critical errors

### ISSUE THREAD

**Issue by jliikala** [TryGhost/Ghost#12493]
Hi!

I'm running Ghost on Digital Ocean droplet (one-click Ghost install). Is this a known (or new) bug or could there be something wrong with my environment on Digital Ocean?

### Issue Summary

Deleting a member via Ghost admin fails with An error occurred Unable to update nested relation. [object Object] error. Further more, if I look through logs, I see two different errors:

NAME: IncorrectUsageError
MESSAGE: i18n error: path "errors.api.userMessages.BookshelfRelationsError" was not found

and

NAME: BookshelfRelationsError
CODE: UPDATE_RELATION
MESSAGE: Unable to update nested relation.

See:

![Screenshot 2020-12-18 at 15 34 01](https://user-images.githubusercontent.com/3062959/102620400-a1066880-4146-11eb-9376-4a6fb15fd1d6.png)

and

![Screenshot 2020-12-18 at 15 41 43](https://user-images.githubusercontent.com/3062959/102620989-96989e80-4147-11eb-8ac6-2221a9489fa2.png)

### To Reproduce

1. Add a new member via Admin manually (or through subscribe form on the Ghost casper theme)
2. Write a test post and send it to members via Ghost email newsletter
3. Go to Members > Member that just got the newsletter
4. Click the Delete member button and confirm delete from the popup
5. Top of the page shows red alert message with text: _An error occurred Unable to update nested relation. [object Object]_

Expected result would be that the member is deleted.

However, if I haven't yet sent the new member a newsletter (Member activity data does not exist yet), deleting the member works fine. So this might have something to do with deleting a member when member activity data exists?

### Technical details:

Running on Digital ocean droplet.

* Ghost Version: 3.40.2
* Node Version: 12.18.0
* Browser/OS: Firefox 83.0 on MacOS Catalina
* Database: MySQL 5.7.32-0ubuntu0.18.04.1 (Ubuntu)

Any help would be much appreciated. Thanks!

**Comment by Taxick** [comment]
Hi :)

I have the same error.. When I try to delete a member on my https://www.mysite.dk/ghost/

I get this error: An error occurred Unable to update nested relation. [object Object]

// Thomas

**Comment by timscha** [comment]
I can confirm this issue with Ghost 3.40.2

Node.js v14.15.2
MariaDB 10.5.8
Ghost-CLI version: 1.15.3

**Comment by mulander** [comment]
I have the exact same issue. You can work around it by connecting to your MySQL database directly and removing the relations manually.

```
mysql> select batch_id from email_recipients where member_email = 'delete@example.org;
5fe8dbdb7819a027d723ffa2
mysql> delete from email_recipients where member_email = 'delete@example.org;
Query OK, 1 row affected (0.00 sec)

mysql> delete from email_batches where batch_id = '5fe8dbdb7819a027d723ffa2';
Query OK, 1 row affected (0.00 sec)
```

After doing these steps you will be able to remove the test user from within the ghost admin interface.

**Comment by Taxick** [comment]
Hi @kevinansfield 

I still get this error when i try to delete a member

Database error, cannot delete member. Unable to update nested relation. [object Object]

Regards

Thomas

**Comment by timscha** [comment]
With 3.40.3 I have the same behavior as @Taxick now.

**Comment by daniellockyer** [comment]
Hey all! 👋🏻 We've just released 3.40.4 which contains the fix for this. Please update your sites using Ghost-CLI 🙂
